### PR TITLE
[feat] Get fan clubs data from back-end

### PR DESCRIPTION
**Pull request by hay92k**
Connect Fan Clubs data to back-end
Change SupportersClub name to FanClub
Refactor SearchListContainer

### CODE REVIEW

File: clients/app/src/utils/helpers/dummy.ts [hunk 1]
```diff
@@ -262,7 +249,7 @@ const generateMatch1 = () => {
     leagueImageURI: generateLeagueURL(),
     matchState: MatcheTimeState.ExtraTime,
     startTime: randomDate(new Date(2021, 10, 10), new Date(2021, 11, 11)),
-    venue: generateSupportersClubName(),
+    venue: 'aa',
```
Comment (rubenaprikyan) [inline clients/app/src/utils/helpers/dummy.ts:252]: what is this ?

Comment (hay92k) [reply clients/app/src/utils/helpers/dummy.ts:252]: this file should be removed very soon.

Comment (rubenaprikyan) [reply clients/app/src/utils/helpers/dummy.ts:252]: oh I see, sorry didn't realized this is a dummy file

File: clients/app/src/views/screens/UIKit/UIKit.tsx [hunk 2]
```diff
@@ -209,15 +209,15 @@ const UIKit: React.FC = () => {
             />
           </View>
           <View style={styles.rowItem}>
-            <SupportersClubCard
+            <FanClubCard
```
Comment (rubenaprikyan) [inline clients/app/src/views/screens/UIKit/UIKit.tsx:212]: This PR is for connect back-end and here is using dummy data, maybe use the back-end data?

Comment (hay92k) [reply clients/app/src/views/screens/UIKit/UIKit.tsx:212]: This usage us in UiKit, thats why I have not change it.

Comment (rubenaprikyan) [reply clients/app/src/views/screens/UIKit/UIKit.tsx:212]: got it

**Review by rubenaprikyan — APPROVED** [review]
lgtm!

**Pull request merged**